src/bridge_gui.cpp

```diff
@@ -130,6 +130,22 @@ class BuildBridgeWindow : public Window {
 		this->SetWidgetDirty(WID_BBS_BRIDGE_LIST);
 	}
 
+	/**
+	* Get the StringID to draw in the selection list and set the appropriate DParams.
+	* @param bridge_data the bridge to get the StringID of.
+	* @return the StringID.
+	*/
+	StringID GetBridgeSelectString(const BuildBridgeData &bridge_data) const {
```

Comment:
```suggestion
	/**
	 * Get the StringID to draw in the selection list and set the appropriate DParams.
	 * @param bridge_data the bridge to get the StringID of.
	 * @return the StringID.
	 */
	StringID GetBridgeSelectString(const BuildBridgeData &bridge_data) const {
```
A bit pedantic, but the first asterisk of each of the lines should be under eachother.